commercetools-models/src/main/java/io/sphere/sdk/projects/CartClassificationDraft.java

```diff
@@ -0,0 +1,14 @@
+package io.sphere.sdk.projects;
+
+
+import io.sphere.sdk.annotations.ResourceValue;
+
+import java.util.Set;
+
+@ResourceValue
```

Comment:
I would expect a `ResourceDraftValue`annotation here and this should allow us to generate the builder classes too. What do you think?

Reply:
Actually when using @ResourceDraftValue to generate a builder for a class with no attribute the generator generates a class with two similar constructor (both are empty) which is not compilable, so basically here the problem is a bug in the generator.
Not sure whether we should create a ticket for this, since a builder for a class with no attribute is not common

Reply:
good catch 👍 
actually i wanted to generate the builder similar to cart score and cart value case, still i agree using the generated model when possible is good 😄